src/Textarea.tsx

```diff
@@ -7,12 +8,42 @@ export interface ITextareaProps extends ITextProps {
   autosize?: boolean;
   minRows?: number;
   maxRows?: number;
-  as?: any;
+
+  value?: string;
+  onChange?: (value: string) => void;
+}
+
+export interface ITextareaState {
+  value?: string;
+}
+
+export class BasicTextArea extends React.Component<ITextareaProps, ITextareaState> {
+  constructor(props: ITextareaProps) {
+    super(props);
+    this.state = { value: this.props.value };
+  }
+
+  private onChange = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+    this.setState({ value: event.target.value });
+
+    if (this.props.onChange) {
```

Comment:
Consider using a default prop here.

Reply:
Might mean defaultProps (like default noop onChange?). I think either way is good, will leave it to ya'll.

Reply:
Yeah, I mean defaultProps, such as noop onChange, but I am fine with both.